app/models/user.rb

```diff
@@ -40,6 +40,8 @@ class User < ApplicationRecord
   SEARCH_CLASS = Search::User
   DATA_SYNC_CLASS = DataSync::Elasticsearch::User
 
+  DELETED_USER = User.new(username: "[deleted user]", name: "[Deleted User]")
```

Comment:
Not sure how I feel about this as a constant here. It's nice bc it is very accessible but at the same time we currently only using this in a view which makes it feel like it should be in a helper. Maybe @vaidehijoshi has some thoughts.

Reply:
It makes me a little nervous to have an actual `User` instance here that we're effectively using as a fallback for deciding how to render a null `User` object within `UserTag` (somehow it feels like it has the potential to be buggy down the road?). I definitely agree that we could be using helpers here, instead!